source/patterns/02-organisms/site-header/site-header.twig

```diff
@@ -0,0 +1,29 @@
+{%- import 'atoms-html' as html -%}
+
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['site-header']) }) -%}
+
+<header {{- html.attributes(attributes) }}>
+
+    {%- set logo = logo|default({}) -%}
+    {%- with logo -%}
+
+        {%- set href = logo.href|default('') -%}
+        {%- set image = logo.image|default({})|merge({src: asset(logo.image.src|default('images/libero-logo.svg')), alt: logo.image.alt|default('Site logo')}) -%}
+
+      {%- block logo -%}
+          {%- if href is defined and href|length > 0 -%}
```

Comment:
I think the logo should always have a link attached to it so it can return to the home page. I know not all users know about this but it has become the normal so think the if statement should be removed in my opinion.

Reply:
Agree the logo _should_ be linked. This check is included as defensive coding to prevent the render from breaking if the link isn't supplied for reasons outside of the pattern library's control.

Reply:
If we can guarantee the `browser` implementation will always send an `href` URL string, we could remove this check. Sound okay @nlisgo?

Reply:
Actually, I'm happy to make it required. Introducing it as optional later will mean it is backwards compatible

Reply:
Rightho. I'll remove the check.